rllib/examples/rock_paper_scissors_multiagent.py

```diff
@@ -156,3 +156,7 @@ def entropy_policy_gradient_loss(policy, model, dist_class, train_batch):
 
     run_with_custom_entropy_loss(args, stop=stop)
     print("run_with_custom_entropy_loss: ok.")
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
@ericl , are you ok with this syntactic change to the example so we don't have to copy paste it?